build/squire-raw.js

```diff
@@ -1099,6 +1099,17 @@ var moveRangeBoundariesDownTree = function ( range ) {
         range.setStart( startContainer, startOffset );
         range.setEnd( endContainer, endOffset );
     }
+
```

Comment:
Fix the cursor missing issue in Firefox after the img element in text is deleted. 
Found this issue when deleting the tab span in Firefox and noticed the img element has the same problem. The tab span will only be used in CEF, so it is not a problem for tab anymore. Just leave this fix here for img since I already got a fix for it.